workspaces/cli-config/src/index.ts

```diff
@@ -234,6 +234,7 @@ export async function createFileTree(config: string, basePath: string) {
       path: gitignorePath,
       contents: `
 captures/
+optic-daemon.log
```

Comment:
I believe to hide any existing file you want ".optic-daemon.log"

Reply:
Didn't mean to make that change, whoops